### ISSUE TITLE
Have english men overtook russians here?

### ISSUE THREAD

**Issue by zaboal** [WinampDesktop/winamp#1082]

**Comment by ArcZ9** [comment]
No, you simple consumed so much Russian that it looks like English now to you.

**Comment by visionavtr** [comment]
нет, мы просто перешли на английский

**Comment by visionavtr** [comment]
хотя некоторые ушли спать

**Comment by arefdsg** [comment]
our land has been reclaimed from the russians

**Comment by ArcZ9** [comment]
> хотя некоторые ушли спать

Stop uttering the black speech of Mordor.

**Comment by zaboal** [comment]
> No, you simple consumed so much Russian that it looks like English now to you.

to consume a language 0_o

**Comment by ArtemBay** [comment]
О нихуя забоал, здарова. Как дела брат?

**Comment by eversiege** [comment]
нихуя вы тут

**Comment by Rorr4** [comment]
The Russians have denied all possibilities of a peace treaty.

**Comment by zaboal** [comment]
> О нихуя забоал, здарова. Как дела брат?

@ArtemBay чет ты с задержкой тут)) пытаюсь всех в матрикс собрать

**Comment by zaboal** [comment]
> The Russians have denied all possibilities of a peace treaty.

@Rorr4 there's so many russians that i can't believe that you are not one of them

**Comment by ArtemBay** [comment]
> > О нихуя забоал, здарова. Как дела брат?
> 
> @ArtemBay чет ты с задержкой тут)) пытаюсь всех в матрикс собрать

Да я только проснулся, решил за завтраком винамап полистать

**Comment by Vukis69** [comment]
Trillions of people should use ruski
